src/storage/src/hummock/store/state_store.rs

```diff
@@ -286,7 +289,7 @@ impl StateStoreIter for HummockStorageIterator {
 
             if iter.is_valid() {
                 let kv = (
-                    Bytes::copy_from_slice(iter.key()),
+                    Bytes::from(iter.key().encode()),
```

Comment:
Can we change the return type of `StateStoreIter` to `(FullKey<Bytes>, Bytes)` instead of the current (Bytes, Bytes) so that we can avoid an encode here and decode in keyspace?

If this change involves too many extra change in unit tests, we can have it in the next PR.

Reply:
Shouldn't it be `FullKey<Vec<u8>>`